internals/secrethub/migrate_config_templates.go

```diff
@@ -48,24 +47,25 @@ func (cmd *MigrateConfigTemplatesCommand) Run() error {
 	return nil
 }
 
-func migrateTemplateTags(inFile io.Reader, outFile io.Writer, mapping referenceMapping, formatString string) ([]string, error) {
+func migrateTemplateTags(inFile io.Reader, outFile string, mapping referenceMapping, formatString string) ([]string, error) {
 	raw, err := ioutil.ReadAll(inFile)
 	if err != nil {
 		return nil, ErrReadFile(inFile, err)
 	}
 
 	var hits, misses []string
 	output := regexpSecretTemplateTags.ReplaceAllStringFunc(string(raw), func(templateTag string) string {
-		path := regexpSecretTemplatePath.FindString(templateTag)
-		if path == "" {
+		path := regexpSecretTemplateTags.FindStringSubmatch(templateTag)
+		fmt.Println(path)
+		if path[1] == "" {
```

Comment:
Can this panic? Should we check the length of `path` variable first?

Reply:
I tested it out and it does not look like it would panic.
I adjusted it, though, for sanity check.
Thanks for pointing it out.

Reply:
Won't panic, but this piece of code could be made a bit more self-documenting. Instead of using the static 1 index everywhere, you could do something like was done here:
https://github.com/secrethub/secrethub-cli/blob/d9dbccfffb571127f28a3ef6a2c7c0650be05e69/internals/secrethub/migrate_config_references.go#L42-L47

Reply:
Also, there's still a print line.

Reply:
The printline was removed with the previous commit: [Prevent regex from panicking](https://github.com/secrethub/secrethub-cli/pull/390/commits/5ba8b592e2ed461931180763efe4abd4c404c1de)

Reply:
Also, nice suggestion. I adjusted the code to be more readable